roles/v2/mining-proxy/proxy-config.toml

```diff
@@ -1,11 +1,11 @@
 upstreams = [
-    { channel_kind = "Group",  address = "127.0.0.1", port = 34254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"},
-    { channel_kind = "Group",  address = "127.0.0.1", port = 44254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"},
-    { channel_kind = "Group",  address = "127.0.0.1", port = 54254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}
+    { channel_kind = "ExtendedWithNegotiator", address = "127.0.0.1", port = 34254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL", jn_values = { address = "127.0.0.1", port = 34264, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}},
+    { channel_kind = "Group", address = "127.0.0.1", port = 44254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL", jn_values = { address = "127.0.0.1", port = 44264, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}},
+    { channel_kind = "Group", address = "127.0.0.1", port = 54254, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL", jn_values = { address = "127.0.0.1", port = 54264, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}},
 ]
-upstreams_jn = [{ address = "127.0.0.1", port = 34264, pub_key = "2di19GHYQnAZJmEpoUeP7C3Eg9TCcksHr23rZCC83dvUiZgiDL"}]
 tp_address = "127.0.0.1:8442" 
 listen_address = "127.0.0.1"
 listen_mining_port = 34255
 max_supported_version = 2
 min_supported_version = 2
+downstream_share_per_minute = 0.01
```

Comment:
Why so low?

Reply:
this is supposed to be used on regetes and with my PC seems a good value. Btw I can change it but not sure about the value to use

Reply:
Ideally the way this would work is you'd specify something like 1 share per minute and the proxy would calculate the correct difficulty to send to the miner such that they submit about 1 share per minute. In my mind .01 shares per minute would mean that the miner will only send 1 share every 100 minutes.

Reply:
Is like that but because when sending `OpenStandardMiningChannel` the test mining-device put a value not related with  the real hash-rate of the cpu (5.4) to make things usable I need to put 0.01 share per minute cause the proxy use that non sense hash rate value to calculate the target. Maybe in the future we should enhance the CPU mining device so that is able to communicate a right hash rate.